packages/test/end-to-end-tests/src/test/loaderTest.spec.ts

```diff
@@ -33,6 +35,26 @@ class TestSharedDataObject1 extends DataObject {
     public get _context() {
         return this.context;
     }
+
+    public async request(request: IRequest): Promise<IResponse> {
+        const url = request.url;
+        const parsed = parse(url, true);
+        const requestParser = RequestParser.create({ url: request.url });
+        const itemId = requestParser.pathParts[0];
+        if (itemId === "bigBlobs") {
```

Comment:
Why do you need to do this? You can just check if parsed.search exists. If yes, return the search params, else call super.request which will handle this part.